docker/build/cudaq.wheel.Dockerfile

```diff
@@ -24,24 +24,13 @@ ARG workspace=.
 ARG destination=cuda-quantum
 ADD "$workspace" "$destination"
 
-RUN installed_versions=$(for python in `ls /usr/local/bin/python*`; do \
-        $python --version | cut -d ' ' -f 2 | egrep -o '^3\.10'; \
-    done | sort -V) && \
-    valid_version=$(for v in $installed_versions; do \
-        comp=$(echo -e "$v\n3.8" | sort -V); \
-        if [ "$comp" != "${comp#3.8}" ]; then echo python$v; fi \
-    done) && \
-    # We need a newer cmake version than what is available with dnf. 
-    dnf remove -y cmake && python3.10 -m pip install cmake; \
-    cd cuda-quantum && \
-    for python in $valid_version; do \
-        echo "Building wheel for $python."; \
-        CUDAQ_BUILD_SELFCONTAINED=ON LLVM_INSTALL_PREFIX="$LLVM_INSTALL_PREFIX" \
-        $python -m build --wheel; \
-        $python -m pip install auditwheel; \
-        LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$(pwd)/_skbuild/lib" \
-        $python -m auditwheel -v repair dist/cuda_quantum-*linux_*.whl; \
-    done
+ARG python_version=3.10
```

Comment:
Have the issues with python 3.10 that you mentioned today been resolved?

Reply:
I am not sure what you referring to. I mentioned running into numpy installation issues with 3.12. My be related https://github.com/numpy/numpy/issues/23808

Reply:
Ah bad memory on my part 🤦 disregard